### DISCUSSION TITLE
Dangerous Trigger a flush of data from ingester to storage

### DISCUSSION THREAD

**Discussion by nottech-gmbh** [grafana/mimir#3067] [Help and support] [answered]
can someone help me , why i get this error? 

<img width="614" alt="image" src="https://user-images.githubusercontent.com/81640282/192609033-e27588c0-1a79-4368-8cce-466af3d19044.png">

my config 

alertmanager:
    data_dir: /data/alertmanager
    external_url: http://mimir.service.mmm.consul:9009/alertmanager
    fallback_config_file: /etc/alertmanager-fallback-config.yaml
    sharding_ring:
        kvstore:
            consul:
                acl_token: '********'
                host: consul.service.
alertmanager_storage:
    backend: s3
    s3:
        access_key_id: 
        bucket_name: mimir-alertmanager
        endpoint: minio-
        http:
            insecure_skip_verify: true
        secret_access_key: '********'
blocks_storage:
    backend: s3
    s3:
        access_key_id: 
        bucket_name: mimir-blocks
        endpoint: minio-
        http:
            insecure_skip_verify: true
        secret_access_key: '********'
    tsdb:
        dir: /data/ingester
compactor:
    cleanup_interval: 5m0s
    sharding_ring:
        kvstore:
            consul:
                acl_token: '********'
                host: consul.service.
            multi:
                mirror_enabled: true
                primary: consul
distributor:
    ha_tracker:
        enable_ha_tracker: true
        kvstore:
            consul:
                acl_token: '********'
                consistent_reads: true
                host: consul.service.
            multi:
                mirror_enabled: true
                primary: consul
    ring:
        kvstore:
            consul:
                acl_token: '********'
                host: consul.service.
            multi:
                mirror_enabled: true
                primary: consul
frontend:
    align_queries_with_step: true
    log_queries_longer_than: 10s
ingester:
    instance_limits:
        max_ingestion_rate: 20000
        max_series: 1500000
        max_tenants: 1000
    ring:
        kvstore:
            consul:
                acl_token: '********'
                host: consul.service.
            multi:
                mirror_enabled: true
                primary: consul
        num_tokens: 512
limits:
    accept_ha_samples: true
    compactor_block_upload_enabled: true
    ingestion_burst_size: 300000
    ingestion_rate: 20000
    out_of_order_time_window: 1m
memberlist:
    join_members:
        - consul.service.mm.consul
querier:
    shuffle_sharding_ingesters_enabled: false
ruler:
    alertmanager_url: http://127.0.0.1:8080/alertmanager
    ring:
        kvstore:
            consul:
                acl_token: '********'
                host: consul.service.
            multi:
                mirror_enabled: true
                primary: consul
    rule_path: /data/ruler
ruler_storage:
    backend: s3
    s3:
        access_key_id:
        bucket_name: mimir-ruler
        endpoint: minio-
        http:
            insecure_skip_verify: true
        secret_access_key: '********'
server:
    http_listen_port: 9009
    log_level: warn
store_gateway:
    sharding_ring:
        kvstore:
            consul:
                acl_token: '********'
                host: consul.service.
            multi:
                mirror_enabled: true
                primary: consul
target: all,alertmanager

**Comment by 56quarters** [comment, marked as answer]
I'm not seeing any error in your screen shot. The "Dangerous" part is just a warning that there are consequences to running the operation. In this case, the link forces the ingester to upload current in memory metrics to object storage.
